mobile/src/main/java/org/openhab/habdroid/model/Widget.kt

```diff
@@ -342,6 +359,10 @@ fun Node.collectWidgets(parent: Widget?): List<Widget> {
         rawMinValue = minValue,
         rawMaxValue = maxValue,
         rawStep = step,
+        row = row,
+        column = column,
+        command = command,
+        releaseCommand = releaseCommand,
```

Comment:
Just set these to `null` and remove the changes from this function. When you're on it: You could add a comment to this function, that it's only used on openHAB versions with XML API, that is openHAB 1 only.